components/router/route.js

```diff
@@ -45,6 +65,9 @@ class GamefaceRoute extends HTMLElement {
      * @param {MouseEvent} event - the event object
      */
     onClick(event) {
+        const activeClass = this.getAttribute('activeClass');
+        if (activeClass) this.classList.add(activeClass);

```

Comment:
We don't need to set the active class on click event because when the link is clicked `onHistoryChange` event will be dispatched and the active class will be changed from the `onHistoryChange` callback.